app/controllers/social_relay_controller.rb

```diff
@@ -0,0 +1,8 @@
+class SocialRelayController < ApplicationController
+  respond_to :json
+
+  def well_known
+    @social_relay = SocialRelayPresenter.new
+    render json: @social_relay
```

Comment:
I don't see any reason for the instance variable here, just pass the instance directly